_deploy/r/gnoswap/common/grc20reg_helper.gno

```diff
@@ -51,9 +51,9 @@ func IsRegistered(path string) error {
 // if token is not registered, it will panic
 func MustRegistered(path string) {
 	if err := IsRegistered(path); err != nil {
-		panic(addDetailToError(
+		panic(newErrorWithDetail(
 			errNotRegistered,
-			ufmt.Sprintf("token(%s)", path),
+			ufmt.Sprintf("token(%s) is not registered", path),
```

Comment:
`errNotRegistered` is value for `[GNOSWAP-COMMON-004] token is not registered`

So, final panic msg for above changes would be...
`[GNOSWAP-COMMON-004] token is not registered token(gno.land/r/abc) is not registered`. 

It contains duplicate string